usaspending_api/bulk_download/v2/views.py

```diff
@@ -21,6 +22,34 @@
 from usaspending_api.bulk_download.models import BulkDownloadJob
 from usaspending_api.download.lookups import JOB_STATUS_DICT
 
+# List of CFO CGACS for list agencies viewset in the correct order, names included for reference
+# TODO: Find a solution that marks the CFO agencies in the database AND have the correct order
```

Comment:
Is this a TODO for this PR or a future sprint?

Reply:
I wrote the TODO as a consideration for a future sprint as it'd modify the database.